WordPress/src/main/java/org/wordpress/android/ui/posts/PostsListFragment.java

```diff
@@ -689,6 +709,32 @@ public void onPostChanged(OnPostChanged event) {
     public void onPostUploaded(OnPostUploaded event) {
         if (isAdded() && event.post.getLocalSiteId() == mSite.getId()) {
             loadPosts(LoadMode.FORCED);
+
+            final PostModel post = event.post;
+            boolean isDraft = post != null && PostStatus.fromPost(post) == PostStatus.DRAFT;
+            if (isDraft) {
+                if (PostUtils.isPublishable(post)) {
+                    if (event.isError()) {
+                        showSnackbar(getString(R.string.editor_draft_saved_locally));
+                    } else {
+                        View.OnClickListener publishPostListener = new View.OnClickListener() {
+                            @Override
+                            public void onClick(View v) {
+                                publishPost(post);
+                            }
+                        };
+                        showSnackbar(R.string.editor_draft_saved_online, R.string.button_publish, publishPostListener);
+                    }
+                }
+            } else {
+                if (event.isError()) {
+                    showSnackbar(getString(R.string.editor_draft_saved_locally));
+                } else {
+                    String message = post.isPage() ? getString(R.string.page_published) :
```

Comment:
addressed in 431094c

side comment: that's a weird condition to have (a `PostUploaded` event without a Post object? sounds more like a FluxC bug if it can at all happen)